streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -83,10 +83,24 @@ public StateStore getStateStore(final String name) {
             throw new StreamsException("Accessing from an unknown node");
         }
 
-        final boolean isGlobalStore = task.topology.globalStateStores().stream()
-            .filter(store -> store.name().equals(name)).count() == 1;
+        final StateStore global = stateManager.getGlobalStore(name);
```

Comment:
Not a comment: this is just to revert base back to trunk until we cleanup GlobalStateManager.